util.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+export default {
+    maybe: maybe,
+    maybeNot: maybeNot
+};
+
+/*
+    impelementation of maybe monad - do action if item is not null
+    item is passed into action callback
+*/
+function maybe(item, action) {
+    if(!item) {
+        return;
+    }
+    return action(item);
+}
+
+/*
+    variation of maybe monad - do action only if item is null
+*/
+function maybeNot(item, action) {
+    if(item) {
+        return;
+    }
+    return action();
+}
```

Comment:
some util functions I was working on for the config/in general that aren't really done